apps/s_server.c

```diff
@@ -1721,6 +1730,11 @@ int s_server_main(int argc, char *argv[])
 #endif
 
 #ifndef OPENSSL_NO_KTLS
+    if (use_zc_sendfile && !use_sendfile) {
+        BIO_printf(bio_out, "Warning: -zerocopy_sendfile depends on -sendfile, enabling -sendfile now.\n");
```

Comment:
This message is not really necessary.

Reply:
I added it to be consistent with:

```
    if (use_sendfile && enable_ktls == 0) {
        BIO_printf(bio_out, "Warning: -sendfile depends on -ktls, enabling -ktls now.\n");
        enable_ktls = 1;
    }
```

and with the documentation that says that `-zerocopy_sendfile` is only valid with `-sendfile` (similar to `-sendfile` and `-ktls`).

Do you suggest that this message isn't needed while the next one (`-sendfile` vs `-ktls`) is still needed? Why?

Reply:
Still not a fan of this message, but it won't block my approval.